yolapy/models/subscription.py

```diff
@@ -0,0 +1,58 @@
+from yolapy.services import Yola
+
+
+class Subscription(object):
```

Comment:
I really like how you implemented this model. Would like to see most of our service models look like this.